Automattic/jetpack
# Mobile Contact Info Block

#### Changes proposed in this Pull Request:

This PR adds `.native.js` equivalents of 5 files related to the jetpack/extensions `contact-info` block in order to make the `contact-info` block usable in the [gutenberg-mobile](https://github.com/wordpress-mobile/gutenberg-mobile) editor used by the [WordPress Android](https://github.com/wordpress-mobile/WordPress-Android) and [iOS apps](https://github.com/wordpress-mobile/WordPress-iOS). 

See related gutenberg-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1934 for integrating the jetpack block into the mobile editor from the jetpack repo. 

![Screenshot_20200409-014328_WordPress](https://user-images.githubusercontent.com/1103838/78876124-b6e25c00-7a03-11ea-8fae-0b455a71574b.jpg)

* This change is only adding `.native.js` versions of existing files in order to enable utilizing the `contact-info` block within the mobile gutenberg editor on Android and iOS (using React Native). 

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Internal reference: p9ugOq-Zb-p2

#### Testing instructions:

* For testing Steps, see Gutenberg-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1934

#### Proposed changelog entry for your changes:

* No change log needed, as the contact info block will initially be hidden from users in the WordPress mobile apps behind a DEV flag.
